client/src/Movies/MovieList.js

```diff
@@ -35,7 +38,7 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
-      <h2>{title}</h2>
+      <Link to={`/movies/${movie.id}`}><button><h2>{title}</h2></button></Link>
```

Comment:
Awesome work adding in the Link component! Note that you could also use `history.push()` to allow the user to be sent to the movie page if they click anywhere on the component